doc/source/user_guide/scale.rst

```diff
@@ -374,5 +374,107 @@ datasets.
 
 You see more dask examples at https://examples.dask.org.
 
+Use Modin
+---------
+
+Modin_ is a scalable dataframe library, which has a drop-in replacement API for pandas and
```

Comment:
"has a drop-in" -> "aims to be a drop-in" would be more accurate

Reply:
@jbrockmendel, could you please elaborate on what gave you this opinion? The user can proceed using pandas API after replacing the import statement.

Reply:
In my experience working on modin lots of behaviors didn't quite match.  Recall I presented a ModinExtensionArray and about half of the tests could not be made to pass because of these.

Reply:
I see, thanks! Yes, "aims to be a drop-in" would be more accurate. Fixed.